portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,69 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
+import com.google.gson.Gson;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*; 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returnd recommendations data*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+ 
+  /**
+  * do request which return as response all the recommendations ap to given limit
```

Comment:
It is better to start function comments with a present simple verb (like Returns), and finish with a .

Please apply to all comments

Reply:
changed in all comments